MatrixKit/Models/MXKAppSettings.m

```diff
@@ -497,7 +497,7 @@ - (void)setEnableBubbleComponentLinkDetection:(BOOL)storeLinksInBubbleComponents
 {
     if (self == [MXKAppSettings standardAppSettings])
     {
-        return [NSUserDefaults.standardUserDefaults objectForKey:@"firstURLDetectionIgnoredHosts"];
+        return [NSUserDefaults.standardUserDefaults objectForKey:@"firstURLDetectionIgnoredHosts"] ?: @[[NSURL URLWithString:kMXMatrixDotToUrl].host];
```

Comment:
If we are registering explicitly an empty array or nil via `setFirstURLDetectionIgnoredHosts` we will always have `@[[NSURL URLWithString:kMXMatrixDotToUrl].host` as result. The question is why we need to do this, do we have an issue with user defaults reset?